### DISCUSSION TITLE
How to upload images for facematch?

### DISCUSSION THREAD

**Discussion by Shamim0018** [vladmandic/human#320] [General]
Hi I am kind of having issues uploading some images for face matching. Is there any UI for Input?

**Comment by vladmandic** [comment]
what kind of problems? what do you mean by ui? for what? this is a library intended to be used in your apps, all included demos are just that - examples.

**Reply by Shamim0018** [reply]
Is there a user interface to add images to the library?

**Comment by vladmandic** [comment]
this is API based library. all UI demos are just demos. 
so no, there is no UI to "add images to the library". and since you need to store images/embeddings somewhere, that would mean pulling in datastore, etc. - and creating an app. Again, that is NOT the goal for `human` - to create end-to-end-app.
